templates/search_clients_daily_v7.sql

```diff
@@ -0,0 +1,153 @@
+WITH
+  augmented AS (
+  SELECT
+    *,
+    ARRAY_CONCAT( ARRAY(
+      SELECT
+        AS STRUCT element.source AS source,
+        element.engine AS engine,
+        element.count AS count,
+        CASE
+          WHEN (element.source IN ('searchbar',  'urlbar',  'abouthome',  'newtab',  'contextmenu',  'system',  'activitystream',  'webextension',  'alias') OR element.source IS NULL) THEN 'sap'
+          WHEN (STARTS_WITH(element.source, 'in-content:sap:')
+          OR STARTS_WITH(element.source, 'sap:')) THEN 'tagged-sap'
+          WHEN (STARTS_WITH(element.source, 'in-content:sap-follow-on:') OR STARTS_WITH(element.source,'follow-on:')) THEN 'tagged-follow-on'
+          WHEN STARTS_WITH(element.source, 'in-content:organic:') THEN 'organic'
+          WHEN STARTS_WITH(element.source, 'ad-click:') THEN 'ad-click'
+          WHEN STARTS_WITH(element.source, 'search-with-ads:') THEN 'search-with-ads'
+        ELSE
+        'unknown'
+      END
+        AS type
+      FROM
+        UNNEST(search_counts.list) ), ARRAY(
+      SELECT
+        AS STRUCT "ad-click:" AS source,
+        key AS engine,
+        value AS count,
+        "ad-click" AS type
+      FROM
+        UNNEST(scalar_parent_browser_search_ad_clicks.key_value) ), ARRAY(
+      SELECT
+        AS STRUCT "search-with-ads:" AS source,
+        key AS engine,
+        value AS count,
+        "search-with-ads" AS type
+      FROM
+        UNNEST(scalar_parent_browser_search_with_ads.key_value) ) ) AS _searches
+  FROM
+    main_summary_v4 ),
+  flattened AS (
+  SELECT
+    *
+  FROM
+    augmented,
+    UNNEST(
+    IF
+      (ARRAY_LENGTH(_searches) = 0,
+        ARRAY(
+        SELECT
+          STRUCT(STRING(NULL) AS source,
+            STRING(NULL) AS engine,
+            NULL AS count,
+            STRING(NULL) AS type)),
+        _searches)) ),
+  -- Aggregate by client_id using windows
+  windowed AS (
+  SELECT
+    ROW_NUMBER() OVER w1_unframed AS _n,
```

Comment:
lets use `submission_date_s3` here as `submission_date`, rather than referencing the parameter at the end
```suggestion
    ROW_NUMBER() OVER w1_unframed AS _n,
    submission_date_s3 AS submission_date,
```